### PR TITLE
commandbox 5.1.0

### PR DESCRIPTION
**Pull request by bdw429s**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by SMillerDev** [comment]
Does this version work with JDK 11 yet?

**Comment by bdw429s** [comment]
@SMillerDev Yes, JDK 11 is so 2 years ago :)  I just tested this version on Java 14 this week.

File: Formula/commandbox.rb [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
   desc "CFML embedded server, package manager, and app scaffolding tools"
   homepage "https://www.ortussolutions.com/products/commandbox"
-  url "https://downloads.ortussolutions.com/ortussolutions/commandbox/5.0.1/commandbox-bin-5.0.1.zip"
-  sha256 "9f855c5b796ac36b010d8b64d58664650d381446d52e0e9167249d4df0269215"
+  url "https://downloads.ortussolutions.com/ortussolutions/commandbox/5.1.0/commandbox-bin-5.1.0.zip"
+  sha256 "cd5a903cc4874930bfd9437e46240f7a980ff7f28c1e136299dc525c47d5ff28"
 
   bottle :unneeded
 
   depends_on :java => "1.8+"
```
Comment (SMillerDev) [inline Formula/commandbox.rb:9]:
```suggestion
  depends_on "openjdk"
```

**Comment by SMillerDev** [comment]
> I just tested this version on Java 14 this week.

Cool, the website only suggested Java 1.7+ though. I suggested an improvement.

**Comment by bdw429s** [comment]
> Cool, the website only suggested Java 1.7+ though.

Yeah, that's a little out of date.  We dropped 1.7 support a few releases ago.  Can you show me the link to where you found that 1.7 reference?  I just did a search and found a couple 1.7 references in my docs, but I'm not sure if I found the one you did.

File: Formula/commandbox.rb (hunk 1 above)
Comment (bdw429s) [reply Formula/commandbox.rb:9]: Since java 1.7 is not actually compatible, only 1.8+ is this suggestion moot?

Comment (SMillerDev) [reply Formula/commandbox.rb:9]: openjdk will always just be the latest java we can provide, which should be used if possible.

**Comment by SMillerDev** [comment]
https://commandbox.ortusbooks.com/setup/requirements#software

File: Formula/commandbox.rb (hunk 1 above)
Comment (bdw429s) [reply Formula/commandbox.rb:9]: Fair enough.  I'll change this now.

**Comment by bdw429s** [comment]
@SMillerDev Suggestion committed and docs updated :)

**Comment by SMillerDev** [comment]
Can you squash both commits into one

**Comment by bdw429s** [comment]
No clue how to do that from the Github web UI.  Don't you have a 'squash and merge' option?  Also, is there a shortage of commit hashes or something? :wink:

**Comment by SMillerDev** [comment]
Normally we can't do squash and merge since it'll skip the bottle flow but I guess here I can (and we're trying to keep the commit history readable by just doing one commit per formula per PR)

**Review by SMillerDev — APPROVED** [review]

**Comment by SMillerDev** [comment]
Thanks @bdw429s ! Without contributions like yours it'd be impossible to keep homebrew going with the high standards that users have come to expect from the project. You can feel good knowing that you've made the world a tiny bit better for homebrew users around the world! 👍 🎉

**Pull request merged**